index.d.ts

```diff
@@ -189,6 +189,13 @@ declare module 'react-universal-component' {
       error:
         | ((p: P) => JSX.Element | ComponentType<P & { error: Error }>)
         | (JSX.Element | ComponentType<P & { error: Error }>);
+    
+      render: (
+        props: P,
+        module: Export | undefined,
```

Comment:
If `Export` is an ES module, this param will be `Export['default']`, not `Export`, due to https://github.com/faceyspacey/react-universal-component/blob/3c21ffa45c7bd9cc9dca5066243ecbcda004ad2a/src/utils.js#L24-L25

Should we do something along these lines?

``` ts
  type ESModuleBase<P> = { default: P };
  // Note: the `default` export is lifted for ES Modules.
  type GetNormalizedModule<M> = M extends ESModuleBase<any> ? M['default'] : M;
```

``` ts
      render: (
        props: P,
        module: GetNormalizedModule<Export> | undefined,
        isLoading: boolean,
        error: Error | undefined
      ) => JSX.Element
```